bluemira/geometry/parameterisations.py

```diff
@@ -1655,7 +1661,7 @@ def create_shape(self, label=""):
         # Bottom leg is flat
 
         bot_leg = PictureFrameTools._make_flat_leg(
-            axis,
+            [0, 1, 0],
```

Comment:
Whats the reasoning behind having these constant instead of using axis?

Reply:
the way the angles are defined - clockwise vs counterclockwise. CP coils have curves in bothe directions, so need to specify fo at least one